src/lv_demo_music/lv_demo_music_main.c

```diff
@@ -58,6 +58,8 @@ static void timer_cb(lv_timer_t * t);
 static void track_load(uint32_t id);
 static void stop_start_anim(lv_timer_t * t);
 static void spectrum_end_cb(lv_anim_t * a);
+int32_t get_cos(int32_t deg, int32_t a);
```

Comment:
Can be they be `static`?

Reply:
Too many commits, too early in the morning, I guess ;)

Reply:
No problem, thanks for this and the other fixes too :slightly_smiling_face: